### ISSUE TITLE
VoiceNextClient missing refrence

### ISSUE THREAD

**Issue by AmbicatusThe2nd** [Emzi0767/DSharpPlus-Example-Bot#4]
Hi, I want my discord bot to stream music into the current voice channel, so I implemented the "join" and "leave" methods which work fine, but I did come across a problem with the configuration VoiceNextConfiguration. When I wanted to implement the property VoiceNextClient it did not show in my suggestions. I doubled checked all the libraries, checked if my nuget packages were up to date and followed the guide in the documentations and installed the dll files. And yet still no class for VoiceNextClient, so I used VoiceNextExtension which worked fine for the configuration but now I do not have the methods that support streaming in the channel like SendSpeakingAsync. How can I fix this or am I missing something?

**Comment by Mehoff** [comment]
For me the problem is even deeper. I have got a weird bot behavior on "leave" command. I have tried writing it by myself, also tried several examples like this official one [https://github.com/DSharpPlus/Example-Bots/tree/master/DSPlus.Examples.CSharp.Ex04](url) 
The error is always the same:
Bot can join to channel, but on "leave" command he always giving me an answer "Not connected in this guild.", even tho he is ;P [https://prnt.sc/stxz1q](url)
I have brazenly copy-pasted code from example above (changed token ofc), but even tho in "Main" cs file 'voice' is defined and not null, inside command 'leave' - VoiceNextConnection is giving me null.
I had never stucked that hard while developing discord bots in node.js.
There cannot be a problem with bot permissions, because I use the same discord bot application with same permissions, that I`ve used before on my node.js bot. I just took the token, and passed it in another project. 
Here is some information for people that have a chance to give me a solution to this problem.
DSharpPlus/CommandsNext/Interactivity/VoiceNext/ v3.2.3,
.Net Core 3.1,
Windows 10,
VS 19,
```
           public VoiceNextClient Voice { get; set; }
...
            var vcfg = new VoiceNextConfiguration
            {
                VoiceApplication = VoiceApplication.Music
            };
            this.Voice = this.Client.UseVoiceNext(vcfg);
```
```
            if (vnc == null)
            {
                //VoiceNextConnection is always null
                await ctx.RespondAsync("Not connected in this guild.");
                return;
            }
```
Thanks for everyone who have read this mess!
